cra-tf-validate-ignore-goals.json

```diff
@@ -1,3 +1,10 @@
 {
-    "scc_goals": []
+    "scc_goals": [
+        {
+            "scc_goal_id": "3000306",
+            "description": " Check whether Event Streams is accessible only by using private endpoints Found in: resource_address: module.event_streams.ibm_resource_instance.es_instance",
```

Comment:
Can we not use private endpoint in the example to get this to pass?

Reply:
Ah ok - so then you need to say that in the `ignore_reason` here instead of pointing to an internal GoldenEye issue (which is now closed). When we create an FSCloud profile example for this module, the CRA scan will be done against that, and that should use private endpoint only. (Tracked at https://github.ibm.com/GoldenEye/issues/issues/3620)